.gitignore

```diff
@@ -1,3 +1,4 @@
+docs
```

Comment:
???

Reply:
Oh, yeah, I'll remove that. I had some notes in my local repository.